clients/src/test/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumerTest.java

```diff
@@ -2182,6 +2207,16 @@ private void completeUnsubscribeApplicationEventSuccessfully() {
         }).when(applicationEventHandler).add(ArgumentMatchers.isA(UnsubscribeEvent.class));
     }
 
+    private void completeAssignmentChangeEventSuccessfully() {
+        doAnswer(invocation -> {
+            AssignmentChangeEvent event = invocation.getArgument(0);
+            // In AsyncKafkaConsumer, the subscription is updated in the background thread.
```

Comment:
nit: this inline comment doesn't seem to add much imo, the func name and body seem clear enough to me